pkg/chaosdaemon/server.go

```diff
@@ -17,6 +17,8 @@ import (
 	"fmt"
 	"net"
 
+	"github.com/pingcap/chaos-mesh/pkg/utils"
```

Comment:
This can be moved with `pb "github.com/pingcap/chaos-mesh/pkg/chaosfs/pb"`

Reply:
Yes. Will fix it.